draft-ietf-quic-transport.md

```diff
@@ -614,10 +634,14 @@ off-path attackers.
 The Initial packet uses long headers with a type value of 0x7F.  It carries the
 first cryptographic handshake message sent by the client.
 
-The client populates the connection ID field with randomly selected values,
-unless it has received a packet from the server.  If the client has received a
-packet from the server, the connection ID field uses the value provided by the
-server.
+If the client has not previously received a Retry packet from the server, it
+populates the Destination Connection ID field with a randomly selected value.
```

Comment:
Is this a single random value for all packets until one is received from teh server? Or is it ok for the client to send a new random in each packet?

Reply:
Oh, I had a whole thing about multiple random values being great for measuring RTT then I remembered that you don't really want to be stuck with trial decryption for the server's first packet.